content/chat/index.adoc

```diff
@@ -0,0 +1,79 @@
+---
+layout: simplepage
+title: Chat
+---
+
+The Jenkins community discusses various project-related topics in multiple channels on the http://www.freenode.net[Freenode] IRC network.
+
+For information on how to connect, see the http://freenode.net/[Freenode] website.
+
+== Channels
+
+=== `#jenkins`
+
+This is the *primary channel* for discussions about Jenkins.
+In you have questions about Jenkins, you'll probably want to ask them here.
+
+http://webchat.freenode.net/?channels=jenkins[Join using a web-based IRC client].
+
+NOTE: if you have a question about Jenkins, just ask it -- don't ask whether you can ask a question.
+It's common to not receive an answer for quite a while if no active contributors are currently checking IRC.
+In that case, just stick around, and periodically (every few hours) ask your question again.
+
+Conversations in this channel are automatically archived to both http://echelog.com/?jenkins[echelog] and https://botbot.me/freenode/jenkins/[botbot.me].
+
+We use the *jenkins-admin* bot to automate common administrative operations.
+Its use is limited to users with _voice_ or _op_ in this channel.
+link:/projects/infrastructure/ircbot/[Learn more].
+"Core" project contributors typically have _op_ (@) in this channel.
```

Comment:
FWIW I got ops a while back to confirm channel logging via botbot.me when both KK and Tyler weren't around a lot. So that was fairly limited in scope and I understand if that is removed again (CC @rtyler ).

That said, it's probably useful to have more than 1-2 people with op around, so the above looks like a sensible suggestion.